src/certificate-roots.toit

```diff
@@ -21,10 +21,10 @@ The most common use-case is to install the common trusted roots
   the $install-all-trusted-roots method.
 
 If you want to minimize the size of the program and memory, you can also
-  selectively install individual roots by calling 'install' on them.
+  selectively install individual roots by calling `install` on them.
 
 Some APIs also accept a list of root certificates, but installing roots
-  for the whole process is typically more convenient.
+  globally is typically more convenient.
```

Comment:
It really isn't global, is it? It is per process.

Reply:
Accidental change due to this file being generated.
https://github.com/toitware/toit-cert-roots/pull/49